docs/migration-guides/prometheus-migration.md

```diff
@@ -0,0 +1,328 @@
+# Prometheus Time Series Database migration
+This article provides a set of guidelines to migrate the time series database (TSDB from now on) from an existing Prometheus deployment to the Bitnami Kubernetes Production Runtime one.
```

Comment:
"This article provides a set of guidelines" sounds like it's more theoretical. I would say "This document describes the steps"

Reply:
done.